packages/webdriverio/tests/commands/browser/url.test.js

```diff
@@ -27,14 +27,39 @@ describe('url', () => {
             .toEqual({ url: 'http://foobar.com/foobar' })
     })
 
-    it('should throw an exception when a non-string value passed in', async () => {
+    it('should throw an exception when a non-string value passed in path', async () => {
         try {
             await browser.url(true)
         } catch (e) {
             expect(e.message).toContain('command needs to be type of string')
         }
     })
 
+    it('should throw an exception when a non-function value passed in inject', async () => {
+        try {
+            await browser.url('/', { inject: 123 })
+        } catch (e) {
+            expect(e.message).toContain('command needs to be type of function')
+        }
```

Comment:
better:

```suggestion
        const err = await browser.url('/', { inject: 123 }).catch((err) => err)
        expect(e.message).toContain('command needs to be type of function')
```

Reply:
`expect(e.message).toContain('command needs to be type of function')` to `        expect(err.message).toContain('command needs to be type of function')
`
I will add this